sdk/eventhub/azure-eventhub-checkpointstoretable/azure/eventhub/extensions/checkpointstoretable/_tablestoragecs.py

```diff
@@ -22,17 +27,217 @@ class TableCheckpointStore:
+                - `partition_id` (str): The partition ID which the checkpoint is created for.
+                - `owner_id` (str): A UUID representing the current owner of this partition.
+                - `last_modified_time` (UTC datetime.datetime): The last time this ownership was claimed.
+                - `etag` (str): The Etag value for the last time this ownership was modified. Optional depending
+                  on storage implementation.
+        """
+        thePartitionKey = "{} {} {} Ownership".format(fully_qualified_namespace,
+        eventhub_name, consumer_group)
+        my_filter = "PartitionKey eq '"+ thePartitionKey + "'"
+        entities = self.table_client.query_entities(my_filter)
+        ownershiplist = []
+        for entity in entities:
+            dic = {}
+            dic[u'ownerid'] = entity[u'ownerid']
+            dic[u'etag'] = entity.metadata.get('etag')
+            dic[u'lastmodifiedtime'] = entity.metadata.get('timestamp')
+            ownershiplist.append(dic)
+        return ownershiplist
+
+    def list_checkpoints(self, fully_qualified_namespace, eventhub_name, consumer_group):
+        """List the updated checkpoints from the storage blob.
+        :param str fully_qualified_namespace: The fully qualified namespace that the Event Hub belongs to.
+         The format is like "<namespace>.servicebus.windows.net".
+        :param str eventhub_name: The name of the specific Event Hub the checkpoints are associated with, relative to
+         the Event Hubs namespace that contains it.
+        :param str consumer_group: The name of the consumer group the checkpoints are associated with.
+        :rtype: Iterable[Dict[str,Any]], Iterable of dictionaries containing partition checkpoint information:
+                - `fully_qualified_namespace` (str): The fully qualified namespace that the Event Hub belongs to.
+                  The format is like "<namespace>.servicebus.windows.net".
+                - `eventhub_name` (str): The name of the specific Event Hub the checkpoints are associated with,
+                  relative to the Event Hubs namespace that contains it.
+                - `consumer_group` (str): The name of the consumer group the checkpoints are associated with.
+                - `partition_id` (str): The partition ID which the checkpoint is created for.
+                - `sequence_number` (int): The sequence number of the :class:`EventData<azure.eventhub.EventData>`.
+                - `offset` (str): The offset of the :class:`EventData<azure.eventhub.EventData>`.
+        """
+        thePartitionKey = "{} {} {} Checkpoint".format(fully_qualified_namespace,
+        eventhub_name, consumer_group)
+        my_filter = "PartitionKey eq '"+ thePartitionKey + "'"
+        entities = self.table_client.query_entities(my_filter)
+        checkpointslist = []
+        for entity in entities:
+            dic = {}
+            dic[u'sequencenumber'] = entity[u'sequencenumber']
```

Comment:
The checkpoint object has the field `sequence_number`. The entity stores the field as `sequencenumber`.
```suggestion
            dic[u'sequence_number'] = entity[u'sequencenumber']
```

Reply:
yes I will push the changes you suggested in this comment